client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -1210,6 +1211,19 @@ static Request putMachineLearningJob(PutJobRequest putJobRequest) throws IOExcep
         return request;
     }
 
+    static Request postMachineLearningOpenJob(OpenJobRequest openJobRequest) throws IOException {
```

Comment:
I'd remove the word `post` from the beginning of this method name.  We don't usually specify the request type when it's POST.

Reply:
Sounds good, I can do that. What do you think of creating our own MlRequestConverter as well? @dimitris-athanasiou  brought it up, and it may be good to get that done sooner rather than later.

Reply:
I was planning to do that as soon as we merge the standing PRs. We'll have to hold on more PRs for a bit, but I think it's worth it :-)